src/network-manager/azext_network_manager/custom.py

```diff
@@ -964,3 +970,210 @@ def network_manager_vnet_security_perimeter_delete(client,
+                         scope_connection_name=scope_connection_name)
+
+
+def network_manager_group_static_member_list(client,
+                                             resource_group_name,
+                                             network_manager_name,
+                                             network_group_name,
+                                             top=None,
+                                             skip_token=None):
+    return client.list(resource_group_name=resource_group_name,
+                       network_manager_name=network_manager_name,
+                       network_group_name=network_group_name,
+                       top=top,
+                       skip_token=skip_token)
+
+
+def network_manager_group_static_member_create(client,
+                                               resource_group_name,
+                                               network_manager_name,
+                                               network_group_name,
+                                               static_member_name):
+    parameters = {}
+    return client.create_or_update(resource_group_name=resource_group_name,
+                                   network_manager_name=network_manager_name,
+                                   network_group_name=network_group_name,
+                                   static_member_name=static_member_name,
+                                   parameters=parameters)
+
+
+def network_manager_group_static_member_update(instance,
+                                               resource_group_name,
+                                               network_manager_name,
+                                               network_group_name,
+                                               static_member_name):
+    return instance
+
+
+def network_manager_group_static_member_show(client,
+                                             resource_group_name,
+                                             network_manager_name,
+                                             network_group_name,
+                                             static_member_name):
+    return client.get(resource_group_name=resource_group_name,
+                      network_manager_name=network_manager_name,
+                      network_group_name=network_group_name,
+                      static_member_name=static_member_name)
+
+
+def network_manager_group_static_member_delete(client,
+                                               resource_group_name,
+                                               network_manager_name,
+                                               network_group_name,
+                                               static_member_name):
+    return client.delete(resource_group_name=resource_group_name,
+                         network_manager_name=network_manager_name,
+                         network_group_name=network_group_name,
+                         static_member_name=static_member_name)
```

Comment:
Don't forget to add an empty line at the end of the file.

```suggestion
                         static_member_name=static_member_name)

```

Reply:
Please fix static analysis CI issues. Thanks.